dev-bin/release.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
```

Comment:
Could we update the README.dev.md given this?

Reply:
Yep, I was going to do that after doing a release so that I could go through the steps myself first.